IFC4x3/Properties/f/FaultPitchAngle_2amUo6QL91DeGgZLQHgMyf/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FaultPitchAngle_2amUo6QL91DeGgZLQHgMyf" Name="FaultPitchAngle" UniqueId="a4c1ec86-6952-4136-842a-8d5691a96f29" PropertyType="p_boundedvalue" PrimaryDataType="IfcPlaneAngleMeasure" />
```

Comment:
Bounded value?

Reply:
just one angle, single value

Reply:
> just one angle, single value

The excel states bounded value in red (with a strikethrough for single value) :-)

Reply:
Changed to single in [28932dd](https://github.com/bSI-InfraRoom/IFC-Specification/pull/600/commits/28932dd0d7ed772eaba167aa06ad026185602c22).